cluster/playbook/roles/common/vars/main.yaml

```diff
@@ -3,3 +3,4 @@ codename: "{{ansible_facts['lsb']['codename']}}"
 kubernetes_package_codename: 'xenial'
 docker_version: "5:19.03.4~3-0"
 containerd_version: "1.2.10-3"
+# kubernetes_version: "1.17.0-00"
```

Comment:
can we have the default value specified in the group var, instead of always want it to be specified by the user?

Reply:
Default value is `latest`. If user wants, they can uncomment the above line and specify the version. Is that okay @jskswamy ?

Reply:
It will be difficult for people to un comment it, can we introduce the variable in group vars?

Reply:
Sure @jskswamy... moved this variable to group_vars.